tests/literals.pyi

```diff
@@ -0,0 +1,5 @@
+from typing import Literal
+
+Literal[None]  # Y061 None inside "Literal[]" expression. Replace with "None"
+Literal[True, None]  # Y061 None inside "Literal[]" expression. Replace with "Literal[True] | None"
+Literal[1, None, "foo", None]  # Y061 None inside "Literal[]" expression. Replace with "Literal[1, 'foo', None] | None"
```

Comment:
This still isn't _great_, because we're suggesting code that we'd immediately complain about, if the user implemented our suggestion. I think you're right, though, that the best way to deal with this would be to introduce another error code that complains about duplicate members inside a `Literal[]` slice -- a version of Y016, but for `Literal[]` slices. We could refrain from emitting Y061 if we knew we'd already emitted that error.

Since this is a really unlikely edge case, I think it would be fine to implement that as a followup PR -- are you interested in doing so? If not, I'm happy to take a look at implementing it :)

Reply:
I agree that a separate check for duplicate elements is probably the cleanest solution here.

I can't guarantee that I'll have the time to work on the followup in the coming days, so if you'd like to have it done sooner rather than later, go for it :)

Reply:
I can't guarantee I'll have time in the next few days either, so I guess we'll just have to see who gets to it first 😄

Reply:
Hey! I hope you can still see the notifications on here :sweat_smile: I saw that you've improved the error message for the duplicate `None` edge case. So I was wondering if we still wanted to add a separate rule for duplicate `Literal[]` members in general?

Reply:
Hiya! Yes, I'd still take that as a PR, I think. We already flag duplicate union members; it seems consistent to also flag duplicate Literal members :)

Reply:
Cool! I'll try to put something together this weekend :)